mlir/lib/Conversion/MathToEmitC/CMakeLists.txt

```
@@ -0,0 +1,19 @@
+add_mlir_conversion_library(MLIRMathToEmitC
+  MathToEmitC.cpp
+  MathToEmitCPass.cpp
+
+  ADDITIONAL_HEADER_DIRS
+  ${MLIR_MAIN_INCLUDE_DIR}/mlir/Conversion/MathToEmitC
+
+  DEPENDS
+  MLIRConversionPassIncGen
+
+  LINK_COMPONENTS
+  Core
+
+  LINK_LIBS PUBLIC
+  MLIREmitCDialect
+  MLIRMathDialect
+  MLIRPass
+  MLIRTransforms
```

```suggestion
  MLIRTransformUtils
```
The header file `"mlir/Transforms/DialectConversion.h"` corresponds to `mlir/lib/Transforms/Utils/DialectConversion.cpp` which is part of the MLIRTransformUtils target, see https://github.com/llvm/llvm-project/blob/7dd34baf5505d689161c3a8678322a394d7a2929/mlir/lib/Transforms/Utils/CMakeLists.txt#L5.

---

Thanks, @marbre. Appreciate your thorough review!